src/components/Todo/Todo.js

```diff
@@ -1,19 +1,28 @@
 import React, { useState } from 'react'
 import ListItem from 'components/ListItem/ListItem'
+import Checkbox from 'components/Checkbox/Checkbox'
 import { FaTrashAlt } from 'react-icons/fa'
 import style from './todo.module.scss'
 
 const Todo = ({ todo, index, handleDeleteIconClick }) => {
   const [visible, setVisible] = useState(false)
   const handleIconDelete = () => setVisible(!visible)
 
+  const [statusCheckbox, setStatusCheckbox] = useState('pending')
+
+  const handleCheckboxClick = (event) => {
+    const status = event.target.checked ? 'completed' : 'pending'
+    setStatusCheckbox(status)
+  }
+
   return (
     <ListItem
-      className={style.listItem}
+      className={`${style[statusCheckbox]}`}
```

Comment:
No hace falta que lo pongas entre comillas porque ya es un string

```suggestion
      className={style[statusCheckbox]}
```